include/core/platform/lf_C11_threads_support.h

```diff
@@ -36,6 +36,12 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <stdlib.h>
 #include <stdint.h> // For fixed-width integral types
 
+// The one and only mutex lock.
+_lf_mutex_t mutex;
```

Comment:
I believe it is considered bad practice to define a global variable in a `.h` file. It should just be declared with the `extern` keyword. Same with the function definitions -- if they are in `.h` they should be `static`, but usually they should be in `.c`